cbmackintosh/rancid-tomatillos
# Fix/final bug fixes

## What does this PR do?
 - Refactors solution for search query persistence to make use of props
 - Adds a page refresh button to the main page error display
 - Improves styling and user feedback on error displays for App and MovieDetailsCard
 - Builds out error handling for VideoSection
 - Makes site header clickable
 - Makes movie poster titles clickable
 - Adds hover state to movie posters

## How should it be tested?
 - Load the app on browser and make sure everything is working as expected. Use Cypress to force network errors to verify error messages are behaving as expected.
